code/modules/mob/living/carbon/brain/brain.dm

```diff
@@ -8,6 +8,7 @@
 	use_me = 0 //Can't use the me verb, it's a freaking immobile brain
 	icon = 'icons/obj/surgery.dmi'
 	icon_state = "brain1"
+	no_vore = TRUE //VOREStation Edit - PLEASE. lol.

```

Comment:
brain_vr.dm?

Reply:
Don't really like making a file for a single line change. It's below my "do I want to complicate things for future mergers, or future code-searchers" threshold.